src/name-resolver.ts

```diff
@@ -96,6 +99,9 @@ export class NameResolver extends Logger {
    * @return     Promise, that resolves to {string} address
    */
   public async getAddress(name: string) {
+    if (!name) {
```

Comment:
Do we need this check here? This function is using `getAddressOrContent`, where we have another check.

Reply:
no need for it. Ran the test without the checks here. Passes.